style.css

```diff
@@ -13,56 +13,133 @@
 body {
   background: #e5e5e5;
   font-family: "Poppins", sans-serif;
-  min-width: 768px;
 }
 
-.body-white {
-  width: 60%;
-  background: #f7f7f9;
+header {
+  padding: 1rem;
+  justify-content: space-between;
+  background-color: white;
 }
 
-.header {
+.header-1 {
   display: flex;
-  width: 27.4375rem;
-  background: white;
+  flex-direction: column;
+  align-content: flex-start;
+  padding: 0 1rem 6rem 1rem;
+  margin: 4rem 1.5rem 2rem 1.5rem;
+}
+
+.header {
+  background-color: white;
   background-image: url(images/bg-image.png);
   background-repeat: no-repeat;
-  background-position: center;
-  padding: 29% 25px;
+  background-size: 100%;
   border-bottom-left-radius: 70px;
-  align-items: center;
+  padding: 5rem 1rem 2rem 1rem;
+
+  /* border: solid green 2px; */
 }
 
-.logo {
+.navbar {
   display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 130%;
 }
 
-.logo a {
+.logo {
   text-decoration: none;
   font-weight: 700;
   font-size: 18px;
   line-height: 16px;
   color: #6070ff;
-  width: 5rem;
+  margin-top: 8px;
 }
 
-.navbar {
-  position: fixed;
+.btn a {
   display: flex;
-  justify-content: space-between;
-  height: 50px;
-  background-color: white;
+  text-decoration: none;
+  color: var(--primary-b-500, #396df2);
+  text-align: center;
+  font-size: 1.3rem;
+  font-weight: 500;
+}
+
+.nav-links li {
+  list-style: none;
 }
 
 .nav-links {
-  display: none;
+  position: fixed;
+  left: -100%;
+  top: 0;
+  flex-direction: column;
+  background: var(--theme-1-primary-colors-2, #6070ff);
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(10px);
+  width: 100%;
+  text-align: center;
+  height: 100vh;
+  padding-top: 4rem;
+}
```

Comment:
- [ ] You improved your mobile menu 👍 according to the previously suggested change, but you can still do better than that. I will suggest you to improve the `text-alignment of the na-link , to stick to the Figma design

Your Design 
![Capture](https://github.com/jerymey/Portfolio-setup/assets/80458199/c8e120bf-9e7f-476c-b52f-543ef0c1edd9)

Figma template

![Capture](https://github.com/jerymey/Portfolio-setup/assets/80458199/3d242b13-90d3-43ee-9fc6-ae3333b369f2)